tests/unit/loader/loader-test.js

```diff
@@ -0,0 +1,208 @@
+QUnit.module("igLoader Common", {
+	errors: [],
+	jsPath: "base/mock/src/js/",
+	cssPath: "base/mock/src/css/",
+
+	beforeEach: function(){
+		this.cleanScripts();
+	},
+
+	resourcesLoaded: function(expectedResources, assert) {
+		//check if expected resources are loaded
+		var scripts = $(document).find("head > script"), actualPaths = [];
+		scripts.each(function () {
+			//console.log("script.src: " + this.src)
+			if (this.src.indexOf("ignite-ui/src/js/") || this.src.indexOf("igniteui/src/js/") > 0) {
+				var array = this.src.split("/");
+				actualPaths.push(array[array.length - 1]);
+			}
+		});
+
+		links = $(document).find("head > link");
+		links.each(function () {
+			//console.log("link.href: " + this.href)
+			if (this.href.indexOf("ignite-ui/src/css/") || this.src.indexOf("igniteui/src/js/") > 0) {
+				var array = this.href.split("/");
+				actualPaths.push(array[array.length - 1]);
+			}
+		});
+
+		for (var i = 0; i < expectedResources.length; i++) {
+			assert.ok(actualPaths.contains(expectedResources[i]), "The loaded resources should contain: " + expectedResources[i]);
+		}
+	},
+	cleanScripts: function() {
+		$(document).find("head > script").each(function () {
+			if (this.src.indexOf("ignite-ui/src/js/") > 0 && this.src.indexOf("src/js/infragistics.loader.js") === -1) {
```

Comment:
Similar to the other check, except this one does check the index and doesn't find anything to remove, leaving the scripts/links in place after the test(s)